<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="GraphView" name="graphicsView">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="sceneRect">
        <rectf>
         <x>-500.000000000000000</x>
         <y>-500.000000000000000</y>
         <width>1000.000000000000000</width>
         <height>1000.000000000000000</height>
        </rectf>
       </property>
       <property name="renderHints">
        <set>QPainter::HighQualityAntialiasing</set>
       </property>
       <property name="dragMode">
        <enum>QGraphicsView::RubberBandDrag</enum>
       </property>
       <property name="cacheMode">
        <set>QGraphicsView::CacheNone</set>
       </property>
       <property name="transformationAnchor">
        <enum>QGraphicsView::AnchorUnderMouse</enum>
       </property>
       <property name="resizeAnchor">
        <enum>QGraphicsView::AnchorUnderMouse</enum>
       </property>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Настройки:</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QToolBox" name="toolBox">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="page_1">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>335</width>
              <height>209</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Список Обьектов</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QListView" name="listView_objects">
               <property name="contextMenuPolicy">
                <enum>Qt::ActionsContextMenu</enum>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::MultiSelection</enum>
               </property>
               <property name="resizeMode">
                <enum>QListView::Fixed</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>335</width>
              <height>98</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Добавить обьекты</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="margin">
              <number>2</number>
             </property>
             <property name="spacing">
              <number>2</number>
             </property>
             <item row="0" column="0">
              <widget class="DragToolButton" name="toolButton">
               <property name="minimumSize">
                <size>
                 <width>40</width>
                 <height>40</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="resourses.qrc">
                 <normaloff>:/images/objects/bricks.png</normaloff>:/images/objects/bricks.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>40</width>
                 <height>40</height>
                </size>
               </property>
               <property name="shortcut">
                <string/>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="autoRepeat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="DragToolButton" name="toolButton_4">
               <property name="minimumSize">
                <size>
                 <width>40</width>
                 <height>40</height>
                </size>
               </property>
               <property name="text">
                <string>Склад</string>
               </property>
               <property name="icon">
                <iconset resource="resourses.qrc">
                 <normaloff>:/images/objects/garage.png</normaloff>:/images/objects/garage.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>40</width>
                 <height>40</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="DragToolButton" name="toolButton_5">
               <property name="minimumSize">
                <size>
                 <width>40</width>
                 <height>40</height>
                </size>
               </property>
               <property name="text">
                <string>Гараж</string>
               </property>
               <property name="icon">
                <iconset resource="resourses.qrc">
                 <normaloff>:/images/objects/factory.png</normaloff>:/images/objects/factory.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>40</width>
                 <height>40</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="DragToolButton" name="toolButton_3">
               <property name="minimumSize">
                <size>
                 <width>40</width>
                 <height>40</height>
                </size>
               </property>
               <property name="text">
                <string>говно</string>
               </property>
               <property name="icon">
                <iconset resource="resourses.qrc">
                 <normaloff>:/images/objects/shit.png</normaloff>:/images/objects/shit.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>40</width>
                 <height>40</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="DragToolButton" name="toolButton_2">
               <property name="minimumSize">
                <size>
                 <width>40</width>
                 <height>40</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="resourses.qrc">
                 <normaloff>:/images/objects/bulldozer.png</normaloff>:/images/objects/bulldozer.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>40</width>
                 <height>40</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Свойства:</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="1">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>X:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Y:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>143</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Позиция:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_pos_x"/>
            </item>
            <item row="1" column="4">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_pos_y"/>
            </item>
            <item row="1" column="5">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Ориентация:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_angle_z"/>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Z:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Имя:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" colspan="5">
             <widget class="QLineEdit" name="lineEdit_name"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Габариты:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Width:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_width"/>
            </item>
            <item row="3" column="3">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Heigth:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="4">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_height"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>648</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_save"/>
    <addaction name="action_load"/>
   </widget>
   <widget class="QMenu" name="menu_map">
    <property name="title">
     <string>Экспорт</string>
    </property>
    <addaction name="action_export"/>
   </widget>
   <widget class="QMenu" name="menu_elements">
    <property name="title">
     <string>Элементы</string>
    </property>
    <addaction name="action_remove"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_map"/>
   <addaction name="menu_elements"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_select_state"/>
   <addaction name="action_drag_scene"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_remove"/>
   <addaction name="action_bring_to_front"/>
   <addaction name="action_send_to_back"/>
  </widget>
  <action name="action_save">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="action_load">
   <property name="text">
    <string>Загрузить</string>
   </property>
  </action>
  <action name="action_export">
   <property name="text">
    <string>Экспорт карты</string>
   </property>
  </action>
  <action name="action_remove">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="action_select_state">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/images/pointer.png</normaloff>:/images/pointer.png</iconset>
   </property>
   <property name="text">
    <string>Выбор</string>
   </property>
  </action>
  <action name="action_drag_scene">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/images/Drag.png</normaloff>:/images/Drag.png</iconset>
   </property>
   <property name="text">
    <string>Перемещение сцены</string>
   </property>
  </action>
  <action name="action_bring_to_front">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/images/bringtofront.png</normaloff>:/images/bringtofront.png</iconset>
   </property>
   <property name="text">
    <string>Перевод на перелний план</string>
   </property>
  </action>
  <action name="action_send_to_back">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/images/sendtoback.png</normaloff>:/images/sendtoback.png</iconset>
   </property>
   <property name="text">
    <string>Отправить на зад</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphView</class>
   <extends>QGraphicsView</extends>
   <header>graphic_scene.graphic_view</header>
  </customwidget>
  <customwidget>
   <class>DragToolButton</class>
   <extends>QToolButton</extends>
   <header>drag_tool_button</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resourses.qrc"/>
 </resources>
 <connections/>
</ui>
